web_hierarchy/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+
+Web Hierarchy
+=============
+This module adds a custom view to the standard odoo 10 views
+whereby users can see a hierarchical chart view of the desired fields.
+It works with two fields; **name** and **parent_id**.
+
+Usage
+=====
+Install module and define a *parent_id* field and a *name* field.
+
+Example
+-------
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+       <odoo>
+        <record id="hr_employee_hierarchy_view" model="ir.ui.view">
+            <field name="name">hr employee hierarchy</field>
+            <field name="model">hr.employee</field>
+            <field name="arch" type="xml">
+                <hierarchy>
+                    <field name="name"/>
+                    <field name="parent_id"/>
+                </hierarchy>
+            </field>
+        </record>
+    </odoo>
+
+.. image:: /images/main_screenshot.png
+    :width: 800px
+    :alt: screenshot
```

Comment:
Screenshot is not working: https://github.com/KKamaa/web/tree/10.0/web_hierarchy

Reply:
this part I don't know why there is no display ...the syntax is ok.

Reply:
I fixed it. you forgot to set static folder.